modules/planning/proto/planning_config.proto

```diff
@@ -18,6 +18,12 @@ enum TaskType {
   POLY_ST_SPEED_OPTIMIZER = 6;
 };
 
+enum NaviTaskType {
```

Comment:
move these task types into TaskType

Reply:
Thank you. 
I thas been modified according to your suggestion.